src/MacroTools/BookSystem/ArtifactSystem/ArtifactBook.cs

```diff
@@ -23,6 +24,14 @@ public ArtifactBook() : base(0.75f, 0.37f, 0.015f, 0.02f)
       Title = "Artifacts";
```

Comment:
Maybe we should rename this to "Artifacts (F7)"?

Reply:
Great call. Done